.github/workflows/app-build-and-deploy.yml

```diff
@@ -0,0 +1,124 @@
+# This workflow will build a docker image, push it to ghcr.io, and deploy it to an Azure WebApp.
+name: Build and Deploy to prod service app
+
+on:
+  workflow_call:
+    secrets: 
+      AZURE_CREDENTIALS:
+        description: 'Service principal that has access to the Azure apps (dev and prod) - Defined in org action secrets'
+        required: true
+      AZURE_WEBAPP_PUBLISH_PROFILE:
+        description: 'Publish profile for the Azure WebApp being deployed - Defined in repo action secrets'
+        required: true
+      AZURE_SECONDARY_WEBAPP_PUBLISH_PROFILE:
+        description: 'Publish profile for a secondary Azure WebApp if being deployed - Defined in repo action secrets'
+        required: false
+      DEPLOY_TOKEN:
+        description: 'Token that is used to determine if the deployment is allowed - Defined in org action secrets'
+        required: true
+      PRODUCTION_DEPLOYERS:
+        description: 'Name of the team that defines who can deploy to production - Defined in org action secrets'
+        required: true
+
+    inputs:
+      deploy-env:
+        description: 'environment to deploy (i.e. dev | prod)'
+        required: true
+        type: string
+      application-type:
+        description: 'application type (i.e. api | worker | ui)'
+        required: true
```

Comment:
`crawler` instead of `worker`?

Reply:
The application type is not named for the repo name.  It is descriptive of what the app does.  Discussed with @qtomlinson and confirmed it's ok to leave as `worker`.